go.mod

```diff
@@ -3,6 +3,9 @@ module github.com/nats-io/nats.go
 go 1.16
 
 require (
+	github.com/golang/protobuf v1.5.2
+	github.com/nats-io/nats-server/v2 v2.9.8
```

Comment:
thanks @cesarvspr for the PR! You can avoid changing both `go.mod` and `go.sum` by running the tests as follows:
```
go test -modfile=go_test.mod ./...
```

Reply:
This works, thanks! Any idea why [tests are intermittent](https://github.com/nats-io/nats.go/runs/9834152071) on the pipeline? @wallyqs